superset-frontend/packages/superset-ui-chart-controls/src/components/Tooltip.tsx

```diff
@@ -46,9 +46,15 @@ export const Tooltip = ({ overlayStyle, color, ...props }: TooltipProps) => {
         overlayStyle={{
           fontSize: theme.typography.sizes.s,
           lineHeight: '1.6',
+          maxWidth: 250,
```

Comment:
If this width is relative to the tooltip's container, maybe we can do something like `calc(100% - gridUnit * 4)` or something?

Reply:
It's not relative to tooltip's container, it's _the_ tooltip's container 🙂 But you're right, let's use grid units